src/main/java/gift/doamin/category/service/CategoryService.java

```diff
@@ -0,0 +1,46 @@
+package gift.doamin.category.service;
+
+import gift.doamin.category.dto.CategoryForm;
+import gift.doamin.category.dto.CategoryParam;
+import gift.doamin.category.entity.Category;
+import gift.doamin.category.exception.CategoryNotFoundException;
+import gift.doamin.category.repository.JpaCategoryRepository;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CategoryService {
+
+    private final JpaCategoryRepository categoryRepository;
```

Comment:
```suggestion
    private final CategoryRepository categoryRepository;
```
이렇게 작성해도 잘 작동되게 해주세요~
스프링의 핵신 기능 DI/IoC를 올바르게 사용하지 않고 계세요.